esmvaltool/diag_scripts/mpqb/mpqb_utils.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+"""Python example diagnostic."""
+import os
+
+import yaml
+
+
+# Get metrics dict, get dataset color(s) or get datasetnames
+def get_mpqb_cfg(cfgtype, cfgkey):
+    """cfgtype: colormap, datasetcolor, datasetname"""
+    # First read cfg file
+    cfg_filename = os.path.join(os.path.split(__file__)[0],
+                                'mpqb_cfg_xch4.yml')
```

Comment:
I think it's better to rename `mpqb_cfg_xch4.yml` -> `mpqb_cfg.yml` and also rename the corresponding file. This part of the diagnostic should not as generic as possible.